dietpi/dietpi-software

```diff
@@ -9419,7 +9419,7 @@ _EOF_
 			G_EXEC chown -R ipfs:ipfs /mnt/dietpi_userdata/ipfs
 
 			# CLI alias
-			echo "alias ipfs='sudo -u ipfs IPFS_PATH=/mnt/dietpi_userdata/ipfs ipfs'" > /etc/bashrc.d/dietpi-ipfs.sh
+			echo "alias ipfs='sudo sudo -u ipfs IPFS_PATH=/mnt/dietpi_userdata/ipfs ipfs'" > /etc/bashrc.d/dietpi-ipfs.sh
```

Comment:
Now there is two times `sudo`, which is not wanted I think? 😄

Ah or you mean that some users may be allowed to call `sudo (-u root)` but not with another target user than root? But that should be better fixed via `sudoers` or by simply running `usermod -aG sudo <username>` instead of cascading `sudo` calls by default. That spams the logs as well and creates nested subshells as well, being slow etc.

Reply:
Is that better?  I'm honestly not sure which patch file to use.